fab/fab/utils.py

```diff
@@ -66,8 +66,8 @@ def tag_commit(self):
 
         pattern = ".*-{}-.*".format(re.escape(self._environment))
         github = _get_github()
-        repo = github.repository('dimagi', 'commcare-hq')
-        for tag in repo.tags(self._max_tags):
+        repo = github.get_organization('dimagi').get_repo('commcare-hq')
```

Comment:
@nickpell fyi this is where the deploy fails with a 400 on ICDS (specifically `get_organization`) Still haven't figured this one out. Tried login/password, and a few different API keys